tests/shared.py

```diff
@@ -95,12 +98,14 @@ def do_entity_test(
         abort_on_first: bool = False,
         profile_identifier: str = DEFAULT_PROFILE_IDENTIFIER,
         rocrate_entity_patch: Optional[dict] = None,
-        rocrate_entity_mod_function: Optional[Callable] = None,
+        rocrate_entity_mod_sparql: Optional[str] = None,
         skip_checks: Optional[list[str]] = ()
 ):
     """
     Shared function to test a RO-Crate entity
     """
+    assert sum([bool(rocrate_entity_patch), bool(rocrate_entity_mod_sparql)]) != 2, "Cannot use rocrate_entity_patch and rocrate_entity_mod_sparql together"
```

Comment:
cleaner option? if I have the logic right
```suggestion
    assert not (rocrate_entity_patch and rocrate_entity_mod_sparql), "Cannot use rocrate_entity_patch and rocrate_entity_mod_sparql together"
```

Reply:
that is definitely a lot cleaner!